docs/execute/binaries.md

```diff
@@ -85,13 +85,9 @@ To run a custom tracer version from a remote branch:
 4. Follow Step 4 from above to run the Parametric tests.
 ## NodeJS library
 
-1. Create a file `nodejs-load-from-npm` in `binaries/`, the content will be installed by `npm install`. Content example:
-    * `DataDog/dd-trace-js#master`
-2. Clone the dd-trace-js repo inside `binaries`
-3. Create a file `nodejs-load-from-local` in `binaries/`, this will disable installing with `npm install dd-trace` and
-   will instead get the content of the file, and use it as a location of the `dd-trace-js` repo and then mount it as a
-   volume and `npm link` to it. For instance, if this repo is at the location, you can set the content of this file to
-   `../dd-trace-js`. This also removes the need to rebuild the weblog image since the code is mounted at runtime.
+1. Clone the dd-trace-js repo inside `binaries`.
```

Comment:
Why removing other options ?

Reply:
Other options are useful for other needs: 

* The `nodejs-load-from-npm` option is very useful if you don't have loccaly the repo, but you need to test a specific branch present in github.
* The `nodejs-load-from-local` option is when your local repo is not in `binaries`, which is the main use case.

Actually, having the repo inside binaries is probably the less used option.